src/telem/lib/input/mekanism/FusionReactorInputAdapter.lua

```diff
@@ -36,16 +36,21 @@ function FusionReactorInputAdapter:beforeRegister ()
         --     environmental_loss       = fn():call('getEnvironmentalLoss'),
         -- },
         coolant = {
+            water                       = fn():call('getWater'):div(1000):fluid(),
             water_capacity              = fn():call('getWaterCapacity'):div(1000):fluid(),
             water_needed                = fn():call('getWaterNeeded'):div(1000):fluid(),
+            steam                       = fn():call('getSteam'):div(1000):fluid(),
             steam_capacity              = fn():call('getSteamCapacity'):div(1000):fluid(),
             steam_needed                = fn():call('getSteamNeeded'):div(1000):fluid(),
         },
         fuel = {
+            tritium                     = fn():call('getTrititum'):div(1000):fluid(),

```

Comment:
Not sure if it is a typo in the mod or here, but I will check in-game later on if `getTrititum` is right. Wouldn't be the first time something like that is real.

Reply:
Looks like it is `getTritium`:

![image](https://github.com/user-attachments/assets/df75cb16-f1c1-4269-874a-db8ad4f398b6)